mamba/mamba/mamba.py

```diff
@@ -747,7 +747,31 @@ def do_call(args, parser):
         from importlib import import_module
 
         module = import_module("conda.cli" + relative_mod, __name__.rsplit(".", 1)[0])
-        exit_code = getattr(module, func_name)(args, parser)
+        if relative_mod == ".main_info":
+            import io
+            from contextlib import redirect_stdout
+
+            with io.StringIO() as buf, redirect_stdout(buf):
+                exit_code = getattr(module, func_name)(args, parser)
+                output = buf.getvalue()
+            # Append mamba version to output before printing/serializing
+            if context.json:
+                from json import loads
+
+                from conda.common.serialize import json_dump
+
+                json_obj = loads(output)
+                json_obj["mamba_version"] = mamba.__version__
+                print(json_dump(json_obj))  # To match conda's format
+            else:
+
+                def format_param(nm, val):
+                    return "%23s : %s" % (nm, val)  # To match conda's format
```

Comment:
Should we convert to an fstring instead?

Reply:
Done!